DietManagerLogger.log

```diff
@@ -50,9 +50,9 @@
   <message>Starting Diet Manager</message>
 </record>
 <record>
-  <date>2020-04-08T12:57:58.529061600Z</date>
-  <millis>1586350678529</millis>
-  <nanos>61600</nanos>
+  <date>2020-04-08T15:49:12.076239300Z</date>
+  <millis>1586360952076</millis>
+  <nanos>239300</nanos>
   <sequence>4</sequence>
   <logger>seedu.dietmanager.commons.core.LogsCentre</logger>
   <level>INFO</level>
```

Comment:
Minor stuff and not important, but was thinking if possible just discard this in your intellij and don't push it up for pr. Cause technically everyone will always keep generating a new one when we run the code.

Reply:
It will generate anyways when we run the checking scripts so I think its pretty harmless

Reply:
Yeah, is harmless, just that if all of us all don't bother to discard and push it for pr, this thing will keep showing a merge conflict. So just a suggestion to be consider for others to avoid having to resolve conflict due to unnecessary stuff like this. The reason why there isn't a merge conflict now is because the rest of us do take note not to push this also